src/components/General/DateAsWords.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div>{{dateInWords(date)}}</div>
+</template>
+
+<script>
+import { mapGetters } from 'vuex'
+import distanceInWordsToNow from 'date-fns/distance_in_words_to_now'
+
+const locales = {
+  en: require('date-fns/locale/en'),
```

Comment:
can you try loading the locales on-demand? we try and keep the bundle sizes as small as possible.

we do it for the app transations here ---> https://github.com/yunity/karrot-frontend/blob/master/src/store/plugins/i18n.js#L10